packages/eslint-config-typescript/README.md

```diff
@@ -62,4 +62,9 @@ rules:
 }
 ```
 
+## Note
+
+For nextjs based project,
+`"@offchainlabs/eslint-config-typescript/react"` should be removed in favor of `"@offchainlabs/eslint-config-typescript/next",`. It includes `react`, `react-hooks`, `jsx-a11y` and `import` configurations already.
```

Comment:
can we include this in README?
useful for setting it up in the bridge ui repo too

https://nextjs.org/docs/basic-features/eslint#rootdir